From da3f45efb970843d5995e3cee290d4f30a32c5e8 Mon Sep 17 00:00:00 2001
From: Swee Yee Fonn <swee.yee.fonn@intel.com>
Date: Mon, 20 Dec 2021 12:57:32 +0800
Subject: [PATCH] SDE/FRP: mixins: add secure_erase_daemon

Add build isecure_erase_daemon from source

Tracked-On: OAM-100444
Signed-off-by: Swee Yee Fonn <swee.yee.fonn@intel.com>
---
 groups/device-specific/caas/AndroidBoard.mk | 7 ++++++-
 1 file changed, 6 insertions(+), 1 deletion(-)

diff --git a/groups/device-specific/caas/AndroidBoard.mk b/groups/device-specific/caas/AndroidBoard.mk
index edc29a2..160d736 100644
--- a/groups/device-specific/caas/AndroidBoard.mk
+++ b/groups/device-specific/caas/AndroidBoard.mk
@@ -31,5 +31,10 @@ rtc-monitor:
 	cd device/intel/civ/host/rtc-monitor && $(LOCAL_MAKE)
 	cp device/intel/civ/host/rtc-monitor/guest_rtc_monitor $(PRODUCT_OUT)/scripts/
 
+.PHONY: secure-erase-daemon
+secure-erase-daemon:
+	cd device/intel/civ/host/secure-erase-daemon && $(LOCAL_MAKE)
+	cp device/intel/civ/host/secure-erase-daemon/secure_erase_daemon $(PRODUCT_OUT)/scripts/
+
 .PHONY: host-pkg
-host-pkg: em-host-utilities vinput-manager rtc-monitor
+host-pkg: em-host-utilities vinput-manager rtc-monitor secure-erase-daemon
-- 
2.17.1

